### ISSUE TITLE
delete or update?

### ISSUE THREAD

**Issue by zoepage** [zoepage/hoodie-drawing#2]
We could update the repo and move it to hoodiehq as far as it's a nice demo app.
What do you think @gr2m @janl @espy ?

**Comment by gr2m** [comment]
yeah no objections at all, it just not a priority at the moment I'd say. But maybe we can reach out to the community to help with that?
